src/main/java/frc/robot/subsystems/auto/AutoAlign.java

```diff
@@ -23,6 +23,16 @@ public static Command autoAlign(CommandSwerveDrivetrain drivebaseSubsystem, Cont
     return new AutoAlignCommand(drivebaseSubsystem, controls).withName("Auto Align");
   }
 
+  public static Command autoAlignL1L(
```

Comment:
Instead of having two routines, have it take a parameter for L or R. and pass it down through the call chain to where it's needed.